docs/_layouts/default.html

```diff
@@ -39,7 +46,20 @@ <h1><a href="{{ "/" | absolute_url }}">{{ site.title | default: site.github.repo
           <li><a href="{{ site.github.repository_url }}">View On <strong>GitHub</strong></a></li>
         </ul>
         {% endif %}
+<!--        <div class="search">
+        <form method="get" id="search" action="https://duckduckgo.com/">
+        <input type="hidden" name="sites" value="smo-wiki.leibniz-hbi.de directly"/>
+        <input class="search" type="text" name="q" maxlength="300" placeholder="Search"/>
+        <input type="submit" value="Search" style="visibility: hidden;" />
+        <button class="button" type="Search">Go</button>
+        </form>
+        </div> -->
         
+          <form action="https://google.com/search" method="get">
```

Comment:
The form should be within in a `<div class="search">` and then give it some more margin via the style.scss
```suggestion
          <div class="search">
          <form action="https://google.com/search" method="get">
```